### DISCUSSION TITLE
Github education support ignoring me?

### DISCUSSION THREAD

**Discussion by pixo2000** [community/community#121543] [GitHub Education]
### Hi everyone,

Hi, i wanted to apply for github education 5 weeks ago. The Problem: The Bot did not approve my application because the image was not ok. I somehow got it to work but then i had another problem: i could not choose my school. Everytime i tried setting it, it didn't save. I contacted support and they answerd. That was 5 Weeks ago. 3 Weeks ago i asked if they were still working on my ticket because they didn't react at all. The only answer i got, was the standart from a bot that they are working on it - 5 weeks ago! What can i do? The documents i provided could provide more than enough proof that im a student.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
